document/presentation/main.tex

```diff
@@ -251,13 +252,25 @@
 \end{tiny}
 
 \begin{itemize}
-\item To implement in your chosen FPGA just type {\tt make fpga}
 \item To implement in your chosen ASIC just type {\tt make asic}
 \item To load the firmware in the hardware just type {\tt make load-firmware}
 \end{itemize}
 
 \end{frame}
 
+
+\begin{frame}{Implement in FPGA}
+\begin{itemize}
+  \item Add your FPGA module in the {\tt ./hardware/fpga} folder
+  \item Add your FPGA board server info in {\tt ./system.mk} 
+  \item To implement it, just type {\tt make fpga}
+  \item To load the FPGA, type {\tt make fpga-load}
```

Comment:
\item To implement and load the hardware design in the FPGA, type {\tt make fpga-load}